examples/trusted_information_retrieval/module/rust/src/lib.rs

```diff
@@ -74,6 +70,57 @@ impl TrustedInformationRetrievalNode {
         // Compute distance.
         earth_radius * central_angle
     }
+
+    /// Create a gRPC client pseudo-Node.
+    fn get_grpc_client() -> Result<DatabaseClient, String> {
+        match oak::grpc::client::Client::new_with_label(
+            &oak::node_config::grpc_client("https://localhost:8888"),
+            &Label::public_untrusted(),
+        )
+        .map(DatabaseClient)
+        {
+            Some(grpc_client) => Ok(grpc_client),
+            None => Err("Couldn't create a gRPC client".to_string()),
+        }
```

Comment:
Does `.or("Couldn't create a gRPC client".to_string())` work here instead of the `match`?